### ISSUE TITLE
Add metric to report active connections/sessions

### ISSUE THREAD

**Issue by mikebin** [confluentinc/ksql#7701]
**Is your feature request related to a problem? Please describe.**
Not a problem, but a requested improvement to observability.

**Describe the solution you'd like**
To better understand and monitor use of a ksqlDB cluster, it would be useful to have metric(s) to report the number of active client connections/sessions, similar to how other databases provide this information. While this might not be especially beneficial for persistent queries (which are detached from a specific user session), it can be helpful for interactive push/pull queries, especially now that pull queries can execute more expensive table scans.

As a follow-on to this, it might be useful to provide a way for administrators to terminate client sessions if needed.

**Comment by vvcephei** [comment]
Hey, thanks for this request, @mikebin !

Matrics would indeed be nice, but I wonder if the thing that DBAs really need is a way to list all the running queries and to be able to terminate them. In my prior experience, we used MySQL, and I used the PROCESSLIST in conjunction with KILL for this purpose (https://www.devart.com/dbforge/mysql/studio/show-running-queries-in-processlist.html is a nice brief overview).

In particular, it's nice to be able to see the client who issued the query, the time the query has been executing, and what the query is actually doing.

**Comment by mikebin** [comment]
Yes, that makes sense, and I think that would provide even more useful functionality than just metric(s).
